<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-6UVCCOR" ignored="0" name="Default test" tests="16" failures="1" timestamp="13 Feb 2019 07:31:20 GMT" time="14.359" errors="0">
  <testcase name="TC001" time="0.626" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC002" time="0.479" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC003" time="0.242" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC004" time="0.43" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC005" time="0.216" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC006" time="0.232" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC007" time="0.607" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC008" time="0.163" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC009" time="4.153" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC010" time="0.175" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC011" time="0.192" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC012" time="1.839" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC013" time="2.141" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC014" time="1.8" classname="freshWorks.AndroidTestNG"/>
  <testcase name="TC015" time="0.827" classname="freshWorks.AndroidTestNG">
    <failure type="java.lang.AssertionError" message="expected [snowboarding] but found [snowboards]">
      <![CDATA[java.lang.AssertionError: expected [snowboarding] but found [snowboards]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at freshWorks.AndroidTestNG.TC015(AndroidTestNG.java:173)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC015 -->
  <testcase name="TC016" time="0.155" classname="freshWorks.AndroidTestNG"/>
</testsuite> <!-- Default test -->
